helpers/create_schema.sql

```diff
@@ -0,0 +1,27 @@
+CREATE DATABASE IF NOT EXISTS your_database_name;
```

Comment:
Strange name of database.

Reply:
This is an auxiliary script that the user can simply run. It was assumed that the name of the database was chosen by the user.